src/main/java/com/buinak/imageapi/controller/ImageController.java

```diff
@@ -37,14 +43,14 @@ public ImageController(ImageService imageService) {
     }
 
     @GetMapping(path = "findImageByName")
-    public ResponseEntity<ImageRepository.ImageInformationView> findImageByName(@RequestParam String name){
-        Optional<ImageRepository.ImageInformationView> optionalImage = imageService.findImageByName(name);
+    public ResponseEntity<Image> findImageByName(String name){
+        Optional<Image> optionalImage = imageService.findImageByName(name);
 
         return optionalImage.map(imageInformationView -> ResponseEntity.ok().body(imageInformationView)).orElseGet(() -> ResponseEntity.notFound().build());
     }
 
     @PatchMapping(path = "patchImage")
-    public ResponseEntity<Optional<ImageRepository.ImageInformationView>> patchImage(@RequestBody Image image){
+    public ResponseEntity<Optional<Image>> patchImage(@RequestBody Image image){
```

Comment:
ResponseEntity of Optional doesn't make sense, return Image, 404 NOT FOUND if not present